techniques/about.html

```diff
@@ -49,73 +49,73 @@
+                  <p><em>テクニック集</em>は、Web Content Accessibility Guidelines (WCAG) を満たす方法の例である。WCAG を満たすための必須要件ではない。WCAG について学び始めるには、<a href="https://www.w3.org/WAI/intro/wcag">WCAG Overview</a> を参照。テクニックに関する重要な情報は、WCAG 解説書の <a href="https://waic.jp/translations/WCAG22/Understanding/understanding-techniques">WCAG 達成基準のテクニックを理解する</a> のセクションを参照。</p>
                 </div>
               </aside>
           <section id="aboutwcag">
-              <h2>About WCAG</h2>
-              <p>Web Content Accessibility Guidelines (WCAG) provides requirements for making websites, applications, and other digital content accessible to people with disabilities. For an introduction to WCAG, supporting technical documents, and educational material, see <a href="https://www.w3.org/WAI/standards-guidelines/wcag/">WCAG 2 Overview</a>.</p>
+              <h2>WCAG について</h2>
+              <p>Web Content Accessibility Guidelines (WCAG) は、ウェブサイト、アプリケーション、及びその他のデジタルコンテンツを、障害のある人にとってアクセシブルにするための要件を提供している。WCAG の概要、関連する技術文書、教材については、<a href="https://www.w3.org/WAI/standards-guidelines/wcag/">WCAG 2 Overview</a> を参照。</p>
 
-          <p>There are additional resources that help you understand and implement WCAG. These Techniques are one type of resource. Others are explained in the <a href="https://www.w3.org/WAI/standards-guidelines/wcag/docs/">WCAG 2 Documents</a>.</p>
+          <p>WCAG を理解し実践するのに役立つ追加のリソースもある。このテクニック集は、そのリソースの一種である。その他のリソースについては <a href="https://www.w3.org/WAI/standards-guidelines/wcag/docs/">WCAG 2 Documents</a> で説明されている。</p>
           </section>
           <section id="introduction">
-            <h2>About Techniques</h2>
-              <p>The techniques provide guidance for web content  authors and evaluators on meeting WCAG success criteria. Techniques are examples of ways to meet WCAG.</p>
+            <h2>テクニック集について</h2>
+              <p>テクニック集は、ウェブコンテンツ制作者及び評価者が WCAG 達成基準を満たすにあたってのガイダンスを提供するものである。テクニック集は、WCAG を満たす方法の例である。</p>
               <section id="updated">
-              <h3>Updated periodically</h3>
-              <p>WCAG 2 itself is a stable document that does not change. The techniques for WCAG are updated periodically to cover more current best practices and changes in technologies and tools.</p>
+              <h3>定期的に更新される</h3>
+              <p>WCAG 2 自身は、変更されることのない安定した文書である。この WCAG テクニック集は、現在のベストプラクティス並びに技術及びツールの変化をより多く扱うために、定期的に更新される文書である。</p>
               </section>
               <section id="not-required">
-              <h3>Techniques are not required</h3>
-              <p>Techniques are informative—that means they are not required. The basis for determining conformance to WCAG 2 is the success criteria from the WCAG 2 standard—not the techniques.</p>
+              <h3>テクニックは必須要件ではない</h3>
+              <p>テクニックは参考情報であり、つまり必須要件ではない。WCAG 2 への適合を判断する基準は、WCAG 2 標準の達成基準であり、テクニックではない。</p>
```

Comment:
このへんは https://www.w3.org/WAI/WCAG21/Understanding/understanding-techniques と同じ文章ですので https://waic.jp/translations/WCAG22/Understanding/understanding-techniques.html の該当箇所と揃えたいです。

揃えるならこう:

```suggestion
              <p>テクニックは、参考情報である。つまり、テクニックは必須要件ではない。WCAG 2 への適合を判断する根拠は、WCAG 2 で規定している達成基準であり、テクニックではない。</p>
```

もし訳を変えるなら解説書の該当箇所もあわせて調整すようにしたいです。